dotnet/src/SemanticKernel.Abstractions/AI/ChatCompletion/ChatHistory.cs

```diff
@@ -9,19 +9,12 @@ namespace Microsoft.SemanticKernel.AI.ChatCompletion;
 /// <summary>
 /// Chat message history representation
 /// </summary>
-public class ChatHistory : List<ChatMessageBase>
+public class ChatHistory : List<ChatMessage>
 {
-    private sealed class ChatMessage : ChatMessageBase
-    {
-        public ChatMessage(AuthorRole authorRole, string content, IDictionary<string, string>? additionalProperties) : base(authorRole, content, additionalProperties)
-        {
-        }
-    }
-
     /// <summary>
     /// List of messages in the chat
     /// </summary>
-    public List<ChatMessageBase> Messages => this;
+    public List<ChatMessage> Messages => this;
```

Comment:
Why is this property necessary? As I noted in https://github.com/microsoft/semantic-kernel/issues/2687, it's strange to me that ChatHistory both derives from `List<ChatMessage>` and exposes a `List<Chat> Messages`; seems like we should have one or the other, but not both.

Reply:
Removed `List<Chat> Messages`

Created  task for the design of `ChatHistory` https://github.com/microsoft/semantic-kernel/issues/3522